src/types/common.ts

```diff
@@ -23,5 +23,5 @@ export interface Tag {
 
 export interface SectionConfig {
   title: string;
-  icon: IconName;
+  icon?: IconName;
```

Comment:
Dlaczego ikonka jest opcjonalna? Z tego co kojarzę, gdybyśmy nie podali ikonki to dana sekcja nie wyświetliłaby się na sidebarze

Reply:
Odpowiedzianie w #55 
https://github.com/KonradSzwarc/devscard/pull/55#discussion_r1002718608